### DISCUSSION TITLE
Large bandwidth usage for signalling channel

### DISCUSSION THREAD

**Discussion by bigbean17** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#2032] [Q&A]
Hi,

I'm using kvs to stream my security camera. I don't really understand why the video source is being streamed even though there is no viewers, this caused a lot of data usage. 

My understanding is that signalling channel is used to find the peer, and viewers will directly connect to the master to view the video, so If there is no viewer, where is the video source being streamed to?

Any help would be appreciated!

**Comment by niyatim23** [comment]
Hi @bigbean17, are you using the unmodified sample application? The SDK sample application does not stream the video if no viewers are connected. Can you share verbose logs demonstrating this?
